pkg/blockatlas/jsonrpc.go

```diff
@@ -41,7 +41,9 @@ func (r *RpcResponse) GetObject(toType interface{}) error {
 	return nil
 }
 
-func (r *Request) RpcCall(result interface{}, method string, params []string) error {
+func (r *Request) RpcCall(result interface{}, method string, params interface{}) error {
+	r.Headers["Content-Type"] = "application/json"
```

Comment:
You can break other implementations. This method is used by all json-rpc calls inside Blockatlas. You can set a custom header inside the platform implementation